src/main/java/picard/sam/util/SamComparison.java

```diff
@@ -72,6 +75,16 @@ public void writeReport(final File output, final List<Header> headers) {
         comparisonMetricFile.write(output);
     }
 
+    public void writeMQConcordanceReport(final File output, final List<Header> headers) {
```

Comment:
does this histogram really need to be written out to a separate file?  Can't it just be added to the main metrics output?